scout/server/blueprints/variant/templates/variant/tx_overview.html

```diff
@@ -35,72 +35,47 @@
         <thead class="thead-light">
           <tr>
             <th>Gene</th>
-            <th>Transcript</th>
+            <th>Refseq IDs</th>
+            <th>ID</th>
             <th>HGVS Description</th>
             <th>Links</th>
           </tr>
         </thead>
         <tbody>
-        {% for gene in variant.genes %}
-          {% if not variant.has_refseq %}
+          {% for gene in variant.genes %}
             {% for transcript in gene.transcripts %}
-              <tr>
```

Comment:
Removing this part, because if the transcript is not primary (has refseq id) is not shown here anyway